TestCases/disc_adj_fsi/dyn_fsi/grad_dv.opt.ref

```diff
@@ -1,9 +1,9 @@
 INDEX	GRAD
-0	-3.465632121754074e-03
-1	-1.844003360219930e-03
-2	-7.925070240684198e-04
-3	-2.742895873196515e-04
-4	-2.738093165742900e-04
-5	-7.890548338696854e-04
-6	-1.831172969426724e-03
-7	-3.431365309796604e-03
+0	-3.461460672772851e-03
+1	-1.841786315630031e-03
```

Comment:
I know the changes are marginal and probably ok, but if this is not a fully converged adjoint and DOT step (not just 20 SU2_CFD_AD steps and then SU2_DOT_AD) I would prefer to have that checked on the 'converged' gradient. Maybe you did this already

Reply:
I changed the settings of the case to make it converge better/faster.